.github/CODEOWNERS

```diff
@@ -1,15 +1,17 @@
 * @nsmithtt
```

Comment:
We were mentioning removing root approvers on F2F. Should we do that now ?
@nsmithtt do you agree ?

Reply:
I would expect that many folks would like to be subscribed to all PRs and getting the same value that you are talking about.
My understanding is that main intention of slack PRs channel #tt-mlir-code-changes was to cover that exact scenario.
Should we fix something with the PRs channel if current implementation is not enough (I know that rest of the team, including me, are using that Slack channel as a source of all PRs)?

Reply:
Would something like this work? In the next UI component, one can check PRs, I _assume_ all PRs would get mailed.

<img width="1425" alt="image" src="https://github.com/user-attachments/assets/11d302ca-01dc-47d9-89fe-4780c877121c">

Reply:
@svuckovicTT, yeah I think that works, much prefer email over the slack channel because the slack channel doesn't have any way to mark which PRs I've reviewed.

Reply:
Should be good now.

Reply:
Thanks! Oh can you also add me to all cmake files & python:
```
*.txt
*.cmake
/python/
```